content/docs/reference/compatibility.md

```diff
@@ -16,29 +16,39 @@ Neon cloud service does not currently provide the user with access permissions o
 
 <a id="default-extensions/"></a>
 
-## Default Extensions
+## Available PostgreSQL Extensions
 
 During technical preview Neon has restrictions on user ability to install PostgreSQL extensions.
```

Comment:
During the technical preview, Neon restricts the ability to install PostgreSQL extensions.